### PR TITLE
updated revision history for release 6.3.1

### PR DESCRIPTION
**Pull request by gr5**

### CODE REVIEW

File: RevisionHistory.html [hunk 1]
```diff
@@ -325,3 +325,20 @@
...
+<li>Fixed a few minor bugs related to "edge mask":
+<ul>
+<li>null calculation when first opening DFTF</li>
+<li>saving edge mask</li>
+<li>improved an edge mask message wording</li>
+</ul>
+</li>
+<li>Improved many things that only affect DFTFringe programming team</li>
+<li>New windows installer</li>
+<li>Linux version of DFTF now exists (only for latest ubuntu) and is always built automatically</li>
```
Comment (atsju) [inline RevisionHistory.html:342]: This is only a suggestion. I'm not native English speaker. Take what you think is best.

As I said in email :
```suggestion
<li>DFTF can now build and run on Linux (build is only tested on latest Ubuntu)</li>
<li>Move to semver format for version names</li>
```

**Comment by gr5** [comment]
I forget that change of text.  Good thing I did a PR, lol.  I merged your suggestion Julien.

**Comment by gr5** [comment]
@githubdoe please approve this review.  I just added notes to the revisionHistory.html file.  No code changes.  I didn't realize this until now but I can't approve my own pull request.  ☹️ Next time I have something this simple I'm going to be tempted to just push it to master.

**Comment by atsju** [comment]
If you can't merge without approval it's because Dale already protected the master branch as I suggested, I think you have no solution here but to wait. You won't be able to push to master I think 

2 possible fix:
- @gr5  should become admin of the repo (then you can tweek the branch protection rule and as an admin you may bypass PR approval)
- add me @atsju as maintainer of whatever rights so that I can also approve PRs. This doubles the chances of somebody available

**Review by githubdoe — APPROVED** [review]

**Comment by githubdoe** [comment]
Need instructions on adding admin for Gr and maintainer for  atsju

**Comment by atsju** [comment]
> Need instructions on adding admin for Gr and maintainer for atsju

I'm looking on how to manage this properly, setting is not exactly were I though. I will keep you updated.

**Pull request merged**